src/Microsoft.Health.Fhir.Core/Features/Operations/Export/Models/ExportJobRecord.cs

```diff
@@ -177,6 +180,9 @@ protected ExportJobRecord()
         [JsonProperty(JobRecordProperties.Parallel)]
         public int Parallel { get; private set; }
 
+        [JsonProperty(JobRecordProperties.Smart)]
+        public bool Smart { get; private set; }
```

Comment:
The same naming convention adopted for the argument in the constructor should be applied here.

Reply:
Fixed.